chef-provisioning-aws.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'chef', '>= 11.16.4'
   s.add_dependency 'chef-provisioning', '~> 1.0'
   s.add_dependency 'aws-sdk-v1', '>= 1.59.0'
```

Comment:
We probably shouldn't do this long run--if people are passing SDK v1 stuff into our APIs, their code will continue to work if we do this, but if we *remove* the dependency their code will break and they will know to fix it.